src/components/search/SimpleSearch.tsx

```diff
@@ -0,0 +1,178 @@
+import {
+  Breadcrumbs,
+  Container,
+  Link,
+  Typography,
+  Grid,
+  Button,
+  Box
+} from '@mui/material'
+import { Formik, Form, FieldArray } from 'formik'
+import SimpleSearchParametersComponent from './SimpleSearchParametersComponent'
+import SearchFlags from './SearchFlags'
+import SimpleSearchDataModel from '../../model/search/SimpleSearchDataModel'
+
+interface Parameters {
+  field: string
+  value: string
+  exact: boolean
+  distance: number
+}
+
+interface Search {
+  parameters: Parameters[]
+}
+enum FlagLabel {
+  ALL_RECORDS = 'ALL RECORDS',
+  GOLDEN_ONLY = 'GOLDEN ONLY',
+  PATIENT_ONLY = 'PATIENT ONLY'
+}
+const Search = () => {
+  const initialValues: Search = SimpleSearchDataModel
+
+  //TODO: set the correct value object type
+  function handleOnFormSubmit(value: Search) {
+    console.log(`send data to backend: ${JSON.stringify(value, null, 2)}`)
+  }
+  return (
+    <Container
+      sx={{
+        ml: 0,
+        mr: 0
+      }}
+      maxWidth={false}
+    >
+      <Grid container direction={'column'}>
+        <Grid item container direction={'row'}>
+          <Grid item lg={6}>
+            <Typography variant="h5">Simple Search</Typography>
+            <Breadcrumbs>
+              <Link underline="hover" color="inherit" href="/search">
+                Search
+              </Link>
+              <Typography color="text.primary">Simple</Typography>
+            </Breadcrumbs>
+          </Grid>
+          <Grid item lg={6}>
+            <SearchFlags
+              button1Label={FlagLabel.ALL_RECORDS}
+              button2Label={FlagLabel.GOLDEN_ONLY}
+              button3Label={FlagLabel.PATIENT_ONLY}
```

Comment:
Same comment as for the ToggleThreeButtons, we pass on an array here `labels={[FlagLabel.ALL_RECORDS, FlagLabel.GOLDEN_ONLY, FlagLabel.PATIENT_ONLY]}`

Reply:
sure